clusterctl/cmd/create_cluster.go

```diff
@@ -78,30 +79,35 @@ func RunCreate(co *CreateOptions) error {
 	if err != nil {
 		return err
 	}
+	ac, err := ioutil.ReadFile(co.AddonComponents)
+	if err != nil {
+		return err
```

Comment:
This error message will be cryptic to a user who doesn't know the codebase, please wrap with something like,

fmt.Errorf("error loading addons file '%v': %v", co.AddonComponents, err)

Reply:
Good catch. I'd just copied the code for provider components from above, so I wrapped that in a nicer error string too.